lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -538,6 +541,17 @@ class SPIRVDecorateBankBitsINTELAttr : public SPIRVDecorate {
   }
 };
 
+class SPIRVDecorateLatencyControlConstraintINTELAttr : public SPIRVDecorate {
```

Comment:
If it's the same as SPIRVDecorateBankBitsINTELAttr, shouldn't we merge API's like:
```
template <SPIRVWord DecorationKind>
SPIRVDecorateWithVectorArg(SPIRVEntry *TheTarget, const std::vector<SPIRVWord> &Args)  : SPIRVDecorate(DecorationKind, TheTarget) { .... };
```
?

Reply:
> 

I modified the code to use SPIRVDecorate directly. And hence this API has been removed. Thanks